queries/html_tags/injections.scm

```diff
@@ -74,3 +74,24 @@
   (#lua-match? @_name "^on[a-z]+$")
   (quoted_attribute_value (attribute_value) @injection.content)
   (#set! injection.language "javascript"))
+
+(attribute
+  ((attribute_name) @_name
+   (#lua-match? @_name "[%[%(].*[%)%]]"))
+  (quoted_attribute_value
+    (attribute_value) @injection.content)
+  (#set! injection.language "angular"))
+
+(attribute
+  ((attribute_name) @_name
+   (#lua-match? @_name "^%*"))
+  (quoted_attribute_value
+    ((attribute_value) @injection.content))
+  (#set! injection.language "angular"))
+
+(element
+  ((text) @injection.content
+   (#lua-match? @injection.content "%{%{.*%}%}")
+   (#offset! @injection.content 0 2 0 -2))
+  (#set! injection.language "angular"))
```

Comment:
This messes with vue interpolations. So a `angular/injections.scm` file is required

Reply:
I was afraid of that... Do you want to just push the necessary changes?

Reply:
Only this queries interfere with vue/react/astro. The other two are fine as is